cmd/guacone/cmd/vulnerability.go

```diff
@@ -50,14 +51,28 @@ type queryOptions struct {
 	vulnerabilityID string
 	depth           int
 	pathsToReturn   int
+	inputType       string
 }
 
 var queryVulnCmd = &cobra.Command{
-	Use:   "vuln [flags] <purl/sbomURI>",
+	Use:   "vuln <type> <input>",
 	Short: "query if a package is affected by the specified vulnerability",
+	Long: `The vuln command allows you to query whether a specific package, SBOM URI, or artifact is affected by a given vulnerability.
+
+Positional Arguments:
+  <type>    Specify the input type: 'artifact', 'uri', or 'purl'
+  <input>   The corresponding input based on the specified type`,
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := logging.WithLogger(context.Background())
 
+		// Ensure exactly two arguments are provided: <type> and <input>
```

Comment:
Are we dropping support for the optional flags in the current version or do they get handled separately from this check?

Reply:
We aren't dropping the support for optional flags, they are still being handled. The only part of the code this PR has changed, is the part for the command line arguments.

https://github.com/guacsec/guac/blob/f292dbcc182b00f6e891c4b6513d081a8bf86a02/cmd/guacone/cmd/vulnerability.go#L580-L587